.github/workflows/main.yaml

```diff
@@ -170,7 +172,7 @@ jobs:
         with:
           tag_name: ${{ steps.get_tag.outputs.git_tag }}
           release_name: ${{ steps.get_tag.outputs.git_tag }}
-          body_path: ./release-artifacts/changelog.md
+          body_path: ./changelog.md
```

Comment:
Would it work with `${{ steps.git-cliff.outputs.changelog }}` ?

Reply:
Lets find out